tests/test_engines.py

```diff
@@ -152,24 +135,6 @@ def test_capture_highest_value_move_captures_highest_value(modified_boards):
         assert str(move) == rec_move
 
 
-def test_capture_highest_value_piece_bishop(minor_boards):
-    """ Tests that bishops capture the highest value piece possible """
-    engine = PrioritizeBishopMoves()
-    board = minor_boards[0][0]
-    rec_move = minor_boards[0][1]
-    move = engine.move(board)
-    assert str(move) == rec_move
-
-
-def test_capture_highest_value_piece_knight(minor_boards):
-    """ Tests that knights capture the highest value piece possible """
-    engine = PrioritizeKnightMoves()
-    board = minor_boards[1][0]
-    rec_move = minor_boards[1][1]
-    move = engine.move(board)
-    assert str(move) == rec_move
-
-
 def test_wrappped_class(versatile_board):
```

Comment:
Can you write the test name such that it's more descriptive when standing alone? One idea may be ```test_prioritize_piece_moves_engine_prioritizes_correct_move```. This is so that if the test fails during CI or just usual running, the user knows exactly what went wrong via. the test name

Reply:
Same goes for the docstring - a slightly more in-depth description of what the test is testing exactly and how it tests it